routes/curations.js

```diff
@@ -7,38 +7,48 @@ const router = express.Router();
 const utils = require('../lib/utils');
 
 // Get a proposed patch for a specific revision of a component
-router.get('/:type/:provider/:namespace/:name/:revision/pr/:pr', asyncMiddleware(async (request, response) => {
-  const coordinates = utils.toEntityCoordinatesFromRequest(request);
-  return curationService.get(coordinates, request.params.pr).then(result => {
-    if (result)
-      return response.status(200).send(result);
-    response.sendStatus(404);
-  });
-}));
+router.get(
+  '/:type/:provider/:namespace/:name/:revision/pr/:pr',
```

Comment:
Probably should ignore most of the router reformatting. They will almost all be changed because of the anonymous function being passed in as the last arg. As separate discussion has raised the point about making named functions for these to aid testing. Probably a good idea.

Reply:
I'd be very cautious about adding anything to this file, but it looks like there's a prettier ignore file if we want to ignore 1 or 2 files here or there https://github.com/prettier/prettier/issues/1428

Reply:
+1 on named functions for testability and composability (when called for, e.g. large-ish functions).

Reply:
we should just "fix" the structure IMHO rather than ignore the files. We'd have to ignore all the routes to get around the issues.